src/components/QuestionRow.jsx

```diff
@@ -0,0 +1,40 @@
+import { decode } from "html-entities";
+
+const QuestionRow = ({
```

Comment:
You seem to be using arrow function components and function declaration components at once, either choose one to use in your whole project but not both.

Reply:
Thanks for the catch